docs/DataModels/DevLakeDomainLayerSchema.md

```diff
@@ -508,6 +508,27 @@ A cicd_task is the abstraction of the bottom-level CI/CD excecution.
 | `finished_date` | datetime        | 3          | When this task finished                                                                 |                   |                  |
 | `cicd_scope_id` | longtext        |            | The id of cicd_scope this task belongs to                                             | FK_cicd_scopes.id |
 
+#### cicd_deployments
+A cicd_deployment is a deployment in a specific repo. It may come from several sources:

```

Comment:
It's not correct, a cicd_deployment is not a deployment in a specific repo. You can refer to the following texts:

A cicd_deployment refers to a deployment at the project level. In the case where a pipeline run or build deploys across three distinct repositories, it will be categorized as ONE cicd_deployment while being recorded as THREE separate [cicd_deployment_commits](url-to-table-cicd_deployment_commits).